### PR TITLE
FM-246 - Implement Aliasing and Optimize assets in FTM

### PR DESCRIPTION
**Pull request by woodsjohn-codev**

### CODE REVIEW

File: src/constants/index.ts [hunk 1]
```diff
@@ -25,64 +25,116 @@ export const EndScene1 = "EndScene1";
...
+export const ASSETS_PATH_FENCE = "./assets/images/fence_v01.webp";
 export const ASSETS_PATH_MONSTER_IDLE = "./assets/images/idle4.png";
-export const DEFAULT_BACKGROUND_1 = "./assets/images/bg_v01.jpg";
-export const HILL_IMAGE_1 = "./assets/images/hill_v01.png";
-export const PILLAR_IMAGE_1 = "./assets/images/Totem_v02_v01.png";
-export const FENCE_IMAGE_1 = "./assets/images/fence_v01.png";
-export const AUTUMN_BACKGROUND_1 = "./assets/images/Autumn_bg_v01.jpg";
+export const DEFAULT_BACKGROUND_1 = "./assets/images/bg_v01.webp";
+export const AUTUMN_BACKGROUND_1 = "./assets/images/Autumn_bg_v01.webp";
 export const AUTUMN_HILL_1 = "./assets/images/Autumn_hill_v01.png";
```
Comment (bernhardccodev) [inline src/constants/index.ts:68]: Question, why is this still in png while the other png are converted to webp?

File: src/scenes/levelend-scene.ts [hunk 2]
```diff
@@ -1,23 +1,22 @@
...
-import { Monster } from "../components/monster";
-import {
-  DEFAULT_BG_GROUP_IMGS,
-  AUTUMN_BG_GROUP_IMGS,
-  WINTER_BG_GROUP_IMGS
-} from '../constants';
+import { loadImages, CLICK } from "@common";
+import { AudioPlayer } from "@components";
+import { CloseButton, NextButton, RetryButton } from "@buttons";
+import { Monster } from "@components";
```
Comment (bernhardccodev) [inline src/scenes/levelend-scene.ts:4]: Line 2 and 4 are both from components

Comment (woodsjohn-codev) [reply src/scenes/levelend-scene.ts]: Thanks @bernhardccodev I applied a fix here.

File: src/constants/index.ts (hunk 1 above)
Comment (woodsjohn-codev) [reply src/constants/index.ts]: @bernhardccodev I updated the asset. thank you for pointing it out.

File: src/constants/index.ts [hunk 3]
```diff
@@ -25,64 +25,116 @@ export const EndScene1 = "EndScene1";
...
+export const MAP_LOCK = "./assets/images/mapLock.png";
+export const STAR = "./assets/images/star.png";
+export const BACK_BUTTON = "./assets/images/back_btn.png";
+export const WIN_BG = "./assets/images/WIN_screen_bg.webp";
+export const PIN_STAR_1 = "./assets/images/pinStar1.png";
+export const PIN_STAR_2 = "./assets/images/pinStar2.png";
+export const PIN_STAR_3 = "./assets/images/pinStar3.png";
+export const CLOUD_6 = "./assets/images/cloud_01.png";
+export const CLOUD_7 = "./assets/images/cloud_02.png";
+export const CLOUD_8 = "./assets/images/cloud_03.png";
```
Comment (bernhardccodev) [inline src/constants/index.ts:108]: Question why are these not in webp format?

File: src/scenes/start-scene.ts [hunk 4]
```diff
@@ -1,17 +1,14 @@
-import { StoneConfig } from "../common/stone-config";
-import { Monster } from "../components/monster";
-import { DataModal } from "../data/data-modal";
-import { Debugger, font, lang } from "../../global-variables";
-import { AudioPlayer } from "../components/audio-player";
+import { StoneConfig, Utils } from "@common";
+import { Monster, AudioPlayer, PlayButton } from "@components";
+import { DataModal } from "@data";
+import { Debugger, font, lang } from "@common";
```
Comment (bernhardccodev) [inline src/scenes/start-scene.ts:4]: Line 1 and 2 are all from common.

File: src/components/pause-popup.ts [hunk 5]
```diff
@@ -1,10 +1,8 @@
...
-import CloseButton from "../components/buttons/close-button";
-import RetryButton from "../components/buttons/retry-button";
-import { CLICK, VISIBILITY_CHANGE } from "../common/event-names";
-import { AudioPlayer } from "./audio-player";
-import AreYouSurePopUp from "./feedback-particle-effect/sure-popup";
-import { lang } from "../../global-variables";
+import { CloseButton, CancelButton, RetryButton } from "@buttons";
+import { CLICK, lang } from "@common";
+import { AudioPlayer } from "@components";
+import { AreYouSurePopUp } from "@components/feedback-particle-effect";
```
Comment (bernhardccodev) [inline src/components/pause-popup.ts:4]: 3 and 4 same path.

File: src/constants/index.ts (hunk 3 above)
Comment (woodsjohn-codev) [reply src/constants/index.ts:108]: I tried. but it become pixelated and its very noticeable. That's why i leave them as is.

**Pull request closed without merging**